### ISSUE TITLE
Wrong initial flex when using react-hot-loader

### ISSUE THREAD

**Issue by panjiesw** [leefsmp/Re-Flex#49]
Hi there, thanks for this cool lib!
I noticed in `ReflexContainer` you use element type comparison to `ReflexSplitter` so that the initial flex calculation is correct when there are 1 or more splitter. 
This is not working correctly when I use react-hot-loader in development, because the elements are _proxified_ (the rendered `ReflexSplitter` element type is now `ProxyComponent`). See gaearon/react-hot-loader#304.

One common solution is to use comparison like this:

```javascript
isSplitterElement (element) {
    return element.type === (<ReflexSplitter/>).type
}
```

I don't know what the implications though by using it like that, but currently that's the only reasonable way in dealing with react-hot-loader. Hope you can fix this!

**Comment by leefsmp** [comment]
Hey, I don't have anything against that workaround but I have no idea if this would hurt performances or not... that function is invoked multiple times during resizing of an element. 

How about the following fix:
```
isSplitterElement (element) {

    if (process.env.NODE_ENV === 'development') {
      //https://github.com/leefsmp/Re-Flex/issues/49
      return element.type === (<ReflexSplitter/>).type
    }

    return element.type === ReflexSplitter
  } 
```

**Comment by panjiesw** [comment]
Hah! That's definitely better! Keep original in the production code, but compromise in development, especially when RHL is involved.

**Comment by leefsmp** [comment]
Ok cool, I published v 2.2.3 with that fix.

**Comment by panjiesw** [comment]
awesome @leefsmp !
